src/machine/machine.h

```diff
@@ -62,6 +62,9 @@ struct Machine {
         int *netif;
         size_t n_netif;
 
+        char *ssh_address;
+        char *ssh_private_key_path;
```

Comment:
if you extend this might make sense to store the VM's CID here too in an extra field. Should be trivial, and so useful.